addons/mail/tests/test_res_partner.py

```diff
@@ -182,3 +182,86 @@ def test_res_partner_merge_wizards(self):
         all_msg = p2_msg_ids_init + p1_msg_ids_init + p1_msg1
         self.assertEqual(len(p2.message_ids), len(all_msg) + 1, 'Should have original messages + a log')
         self.assertTrue(all(msg in p2.message_ids for msg in all_msg))
+
+    def test_res_partner_get_starred_count(self):
+        # ignored message has (according to is_empty):
+        # - empty body OR
+        # - no subtype description OR
+        # - no tracking_value_id
```

Comment:
as suggested above to resolve dependency potential issue, probably split into multiple tests, one per "setup" that doesn't require changing the message afterwards (this is also much easier to maintain later on, as each test shouldn't influence each other)